config/site_settings.yml

```diff
@@ -2039,6 +2039,13 @@ developer:
     type: tag_list
     default: ""
     client: true
+  default_sidebar_topic_destination:
+    hidden: true
```

Comment:
I see, keeping it hidden for now! looks good to me, and aligns with how we do other settings.